### ISSUE TITLE
Upgrading to 1.3.8 caused: sql: Scan error on column index 5, name "condition_name": converting NULL to string is unsupported

### ISSUE THREAD

**Issue by jamesharv** [openfga/openfga#1211]
**Summary**

Immediately after upgrading to 1.3.8 from 1.3.7 we started getting internal server errors. Rolling back to 1.3.7 resolved the issue. In the logs we found the error shown below.

**OpenFGA version or commit**

Immediately after upgrading to 1.3.8

**Store data**
-

**Other data**

- docker on AWS ECS
- Postgres
- Are you running OpenFGA with any configuration overrides or with any of the flags mentioned in `./openfga run --help`?
- Logs:

Example check request:
```
2023-12-05T01:11:41.242Z	ERRORrpc error: code = Code(4000) desc = Internal Server Error
{"grpc_service": "openfga.v1.OpenFGAService", "grpc_method": "Check", "grpc_type": "unary", "request_id": "e27dc9cb-d13a-4537-b375-b14fca90fb9a", "user_agent": "openfga-sdk js/0.2.10", "raw_request": {"store_id":"01H92CEC8HDM58GWGEBRWPQHSS","tuple_key":{"user":"user:7fd5822a-5e9d-46cb-b97d-f8669d30ab6e","relation":"manager","object":"site:0a5ecb2e-f9b0-463c-a3b1-df0886465bbd"},"contextual_tuples":{"tuple_keys":[]},"authorization_model_id":"01HGSC2RJ3XT4BM1Q27MH50HC8","trace":false,"context":null}, "raw_response": {"allowed":false,"resolution":""}, "peer.address": "127.0.0.1:51082", "store_id": "01H92CEC8HDM58GWGEBRWPQHSS", "authorization_model_id": "01HGSC2RJ3XT4BM1Q27MH50HC8", "grpc_code": 4000, "internal_error": "sql: Scan error on column index 5, name \"condition_name\": converting NULL to string is unsupported"}
```

Example read request:
```
2023-12-05T01:12:16.355Z	ERROR	rpc error: code = Code(4000) desc = Internal Server Error
{"grpc_service": "openfga.v1.OpenFGAService", "grpc_method": "Read", "grpc_type": "unary", "request_id": "12ea7e39-46b7-4480-9b2c-6fde38a61777", "user_agent": "openfga-sdk js/0.2.10", "raw_request": {"store_id":"01H92CEC8HDM58GWGEBRWPQHSS","tuple_key":{"user":"user:8b1ab6bf-bad7-454a-bb4a-c552ff60a6af","relation":"admin","object":"region:US"},"page_size":1,"continuation_token":""}, "raw_response": {"tuples":[],"continuation_token":""}, "peer.address": "127.0.0.1:51082", "store_id": "01H92CEC8HDM58GWGEBRWPQHSS", "grpc_code": 4000, "internal_error": "sql: Scan error on column index 5, name \"condition_name\": converting NULL to string is unsupported"}
```

**Comment by jpadilla** [comment]
@jamesharv thanks for the report, working on a fix and added test coverage on #1212

**Comment by jon-whit** [comment]
@jamesharv the issue should be resolved in the `v1.3.9` release. Could you please upgrade and confirm?
